src/plots/plot.ts

```diff
@@ -465,7 +484,9 @@ export class Plot extends Component {
           index: datasetIndex,
           dataset: dataset,
           position: position,
-          selection: drawer.selectionForIndex(validDatumIndex),
+          drawer: drawer,
+          validDatumIndex: validDatumIndex,
+          selection: null,
```

Comment:
The fact that `selection` is null here sounds really mysterious.  Why can't this attribute be just defined instead?